angularjs-portal-frame/src/main/webapp/css/buckyless/portlet-frame.less

```diff
@@ -0,0 +1,50 @@
+/* Portlet Frame */
+
+/*  
+The portlet frame is the div that houses the main content of the page. 
+It contains a header div (.portlet-header), usually with a title, description, and background image. 
+It also contains a body div (.portlet-body), that houses the content.
+*/
+
+.portlet-frame {
```

Comment:
Do we want to wrap the whole `portlet-frame.less` file in a namespace. Say .portlet-frame?

Therefore we for sure know that it will only effect things in the .portlet-frame scope?  

#TimRandomThoughts